ush/plotObsMon.py

```diff
@@ -118,6 +123,7 @@ def loadConfig(satname, instrument, obstype, plot, cycle_tm, cycle_interval,
     Example calling sequence: >python plotObsMon.py -i ../parm/gfs/gfs_plots.yaml -p 2023122000
     """
 
+    start = subprocess.check_output('date')
```

Comment:
is this just to get timing information? if so, you should use the internal python `datetime` module.

```
from datetime import datetime, timezone
start = datetime.now(timezone.utc)
...
end = datetime.now(timezone.utc)
```

Reply:
Yes that was for timing info.  Honestly it could probably just be removed now -- I just wanted an idea of the wall time for the job(s).  I'll go ahead and rm it.